feedthefox/users/adapters.py

```diff
@@ -64,7 +64,12 @@ def populate_user(self, request, sociallogin, data):
                 if mozillian[attr].get('privacy') == 'Public':
                     setattr(user, attr, mozillian[attr].get('value'))
             if mozillian['full_name']['privacy'] == 'Public':
-                first_name, last_name = mozillian['full_name']['value'].split(' ', 1)
+                full_name = mozillian['full_name']['value'].split(' ', 1)
+                try:
+                    first_name, last_name = full_name
+                except ValueError:
+                    first_name = full_name[0]
```

Comment:
full_name is a required field in mozillians.org so there should be always at least a character.